session/portforward.go

```diff
@@ -169,22 +168,36 @@ func (f *MenderPortForwarder) Write(body []byte) error {
 	return nil
 }
 
+type PortForwardHandler struct {
+	portForwarders map[string]*MenderPortForwarder
+}
+
 func PortForward() Constructor {
-	f := HandlerFunc(portForwardHandler)
-	return func() SessionHandler { return f }
+	return func() SessionHandler {
+		return &PortForwardHandler{
+			portForwarders: make(map[string]*MenderPortForwarder),
+		}
+	}
+}
+
+func (h *PortForwardHandler) Close() error {
+	for _, f := range h.portForwarders {
+		f.Close(true)
```

Comment:
This function is called after a session close condition and no messages should be sent nor received.
```suggestion
		f.Close(false)
```

Reply:
right, sorry, I did not remember it